contrib/caddy/caddyfile

```diff
@@ -0,0 +1,6 @@
+cinny.domain.tld {
+  @nativeRouter not file {path} /
+  rewrite @nativeRouter {http.matchers.file.relative}
+  root * /path/to/caddy/dist
+  file_server
+}
```

Comment:
Hi, I am not familiar with caddy, but could you please clarify that,

Does this also set redirects for files like [the ngnix](https://github.com/cinnyapp/cinny/blob/b7e5e0db3e135534b33896f5de139ef8331f1e30/contrib/nginx/cinny.domain.tld.conf#L21-L33) file does?

Reply:
Lets keep it simple then, thanks for explaining.